.storybook/preview.js

```diff
@@ -13,7 +14,25 @@ addDecorator((storyFn) => (
   </>
 ));
 
+/**
+ * We need to provide our theme to evert page in storybook.
+ * Before we used a story decorator only, but that's not enough
+ * as it won't provide any styles to pages without stories.
+ */
+const DocsPageContainer = ({ children, context }) => {
+  return (
+    <DocsContainer context={context}>
+      <ThemeProvider theme={theme}>
+        <GlobalStyle $lightTheme={true} />
+        {children}
+      </ThemeProvider>
+    </DocsContainer>
+  );
+};
```

Comment:
plain mdx files without a story won't have our global styles applied. This is the fix👍